DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -126,6 +126,23 @@ private function sessionLoad($config, ContainerBuilder $container)
                 case 'jackrabbit':
                     if (empty($loaded['jackalope'])) {
                         $this->loader->load('jackalope.xml');
+
+                        $jackalopeTransports = array('prismic', 'doctrinedbal', 'jackrabbit');
+                        foreach ($jackalopeTransports as $transport) {
+                            $factoryServiceId = sprintf('doctrine_phpcr.jackalope.repository.factory.service.%s', $transport);
+                            $factoryService = $container->getDefinition(sprintf('doctrine_phpcr.jackalope.repository.factory.%s', $transport));
```

Comment:
this is not the factory service, but the service built using the factory. The naming is confusing